amd_openvx_extensions/amd_rpp/source/ResizeMirrorNormalizeTensor.cpp

```diff
@@ -0,0 +1,337 @@
+	data->srcBatch_height = (Rpp32u *)malloc(sizeof(Rpp32u) * data->nbatchSize);
+	data->dstBatch_width = (Rpp32u *)malloc(sizeof(Rpp32u) * data->nbatchSize);
+	data->dstBatch_height = (Rpp32u *)malloc(sizeof(Rpp32u) * data->nbatchSize);
+	data->dstImgSize = (RpptImagePatch *)malloc(sizeof(RpptImagePatch) * data->nbatchSize);
+
+    STATUS_ERROR_CHECK(vxQueryImage((vx_image)parameters[0], VX_IMAGE_HEIGHT, &data->maxSrcDimensions.height, sizeof(data->maxSrcDimensions.height)));
+    STATUS_ERROR_CHECK(vxQueryImage((vx_image)parameters[0], VX_IMAGE_WIDTH, &data->maxSrcDimensions.width, sizeof(data->maxSrcDimensions.width)));
+    data->maxSrcDimensions.height = data->maxSrcDimensions.height / data->nbatchSize;
+    STATUS_ERROR_CHECK(vxQueryImage((vx_image)parameters[3], VX_IMAGE_HEIGHT, &data->maxDstDimensions.height, sizeof(data->maxDstDimensions.height)));
+    STATUS_ERROR_CHECK(vxQueryImage((vx_image)parameters[3], VX_IMAGE_WIDTH, &data->maxDstDimensions.width, sizeof(data->maxDstDimensions.width)));
+    data->maxDstDimensions.height = data->maxDstDimensions.height / data->nbatchSize;
+
+    // Check if it is a RGB or single channel U8 input
+    vx_df_image df_image = VX_DF_IMAGE_VIRT;
+    STATUS_ERROR_CHECK(vxQueryImage((vx_image)parameters[0], VX_IMAGE_ATTRIBUTE_FORMAT, &df_image, sizeof(df_image)));
+    uint ip_channel = (df_image == VX_DF_IMAGE_RGB) ? 3 : 1;
+
+    // Initializing tensor config parameters.
+    data->srcDescPtr = &data->srcDesc;
+    data->dstDescPtr = &data->dstDesc;
+
+    data->srcDescPtr->dataType = RpptDataType::U8;
+    data->dstDescPtr->dataType = RpptDataType::U8;
+
+    // Set numDims, offset, n/c/h/w values for src/dst
+    data->srcDescPtr->numDims = 4;
+    data->dstDescPtr->numDims = 4;
+    data->srcDescPtr->offsetInBytes = 0;
+    data->dstDescPtr->offsetInBytes = 0;
+    data->srcDescPtr->n = data->nbatchSize;
+    data->srcDescPtr->h = data->maxSrcDimensions.height;
+    data->srcDescPtr->w = data->maxSrcDimensions.width;
+    data->srcDescPtr->c = ip_channel;
+    data->dstDescPtr->n = data->nbatchSize;
+    data->dstDescPtr->h = data->maxDstDimensions.height;
+    data->dstDescPtr->w = data->maxDstDimensions.width;
+    data->dstDescPtr->c = ip_channel;
+   
+	data->srcDescPtr->layout = RpptLayout::NHWC;
+	data->dstDescPtr->layout = RpptLayout::NHWC;
+
+	data->srcDescPtr->strides.nStride = ip_channel * data->srcDescPtr->w * data->srcDescPtr->h;
+	data->srcDescPtr->strides.hStride = ip_channel * data->srcDescPtr->w;
+	data->srcDescPtr->strides.wStride = ip_channel;
+	data->srcDescPtr->strides.cStride = 1;
+
+	data->dstDescPtr->strides.nStride = ip_channel * data->dstDescPtr->w * data->dstDescPtr->h;
+	data->dstDescPtr->strides.hStride = ip_channel * data->dstDescPtr->w;
+	data->dstDescPtr->strides.wStride = ip_channel;
+	data->dstDescPtr->strides.cStride = 1;
+
+
+    // Initialize ROI tensors for src/dst
+    data->roiTensorPtrSrc  = (RpptROI *) calloc(data->nbatchSize, sizeof(RpptROI));
+
```

Comment:
@SundarRajan28 Please look into all these codacy erros

Reply:
Fixed the codacy warnings in the new commit